src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -55,6 +61,30 @@
                 </button>
             </template>
         </modal>
+        <modal :open="projectToCopy || versionToCopy">
+            <h4 v-if="projectToCopy" v-translate="'copyProjectHeader'"></h4>
+            <h4 v-if="versionToCopy" v-translate="'copyVersionHeader'"></h4>
+            <h5 v-translate="'enterProjectName'"></h5>
+            <template v-slot:footer>
+                <div class="container">
+                    <div class="row">
+                        <input type="text" class="form-control" v-translate:placeholder="'projectName'" v-model="copiedProjectName">
+                    </div>
+                    <div class="row">
+                        <button type="button"
+                            class="btn btn-white mt-2 mr-1 col"
```

Comment:
I see that you've used `btn-white` for the OK action and `btn-red` for the cancel action because those are what's used in the delete modal. In fact this is backwards since `btn-red` is our primary call to action button (that one must have slipped through code review!), so both these modals should have the button colours swapped. Do you mind fixing that for both modals in this branch?